sentry_sdk/utils.py

```diff
@@ -43,18 +24,8 @@
     # Python 3.10 and below
     BaseExceptionGroup = None  # type: ignore
 
-from datetime import datetime
-from functools import partial
-
-try:
-    from functools import partialmethod
```

Comment:
`partialmethod` [was added in 3.4](https://docs.python.org/3/library/functools.html#functools.partialmethod), so we'll always have it now